Dnn.CommunityForums/module.css

```diff
@@ -1,800 +1,1361 @@
-.afgridhd{font-weight:bold;text-align:center !important; background-color:#666;color:#fff !important;}
-.afSecGrid #hd1 .sec1{background-color:#FFFF99;color:#666 !important;border:solid 1px #aaa; padding:0px}
-.afSecGrid #hd2 .afgridhdsub{background-color:#f5f5f5;color:#666 !important;text-align:center;padding-top:0px;padding-bottom:0px;}
-.afSecGrid #hd2 .colstart{border-left:solid 1px #aaa;}
-.afSecGrid #hd2 .colend{border-right:solid 1px #aaa;}
-.gridsep{border-right:solid 1px #aaa}
-.afSecGrid #row1 td{border-left:solid 1px #aaa;border-top:solid 1px #aaa;}
-.afSecGrid #row2 td{border-left:solid 1px #aaa;border-top:solid 1px #aaa;}
-.afSecGrid #row3 td{border-left:solid 1px #aaa;border-top:solid 1px #aaa;}
-.afSecGrid #row4 td{border-left:solid 1px #aaa;border-top:solid 1px #aaa;border-bottom:solid 1px #aaa;}
-.afSecGrid #row1 .colend{border-right:solid 1px #aaa;}
-.afSecGrid #row2 .colend{border-right:solid 1px #aaa;}
-.afSecGrid #row3 .colend{border-right:solid 1px #aaa;}
-.afSecGrid #row4 .colend{border-right:solid 1px #aaa;}
-.afSecGrid .gridcheck{text-align:center !important;}
-.afSecGrid .gridcheck input{padding:0;margin:0;margin:4px;}
-.afSecGrid .rowhd{font-weight:bold;padding-right:4px;background-color:#f5f5f5;}
-
-.dnnForm .dnnFormItem .afavatarform input{width:55px;min-width:inherit; float:none;padding:1px}
-.afavatarform li{display:inline;margin-right:8px; list-style-type:none;}
-
-.af-colstats_responsive,
-.af_lastpost_responsive {
-	display: none;
-    white-space:nowrap;
-}
+	.af-fileicon-sxd {
+		background-position: -224px -32px;
+	}
 
+	.af-fileicon-sxi {
+		background-position: -256px -32px;
+	}
 
-@media only screen and (max-width: 768px), 
-only screen and (max-device-width: 768px) {
-.af-lastpost,
-.af-colstats,
-.afgrouprow-m,
-.afgrouprow-l {
-	display: none;
-}
+	.af-fileicon-sxw {
+		background-position: -288px -32px;
+	}
 
-.af-colstats_responsive {
+	.af-fileicon-tar {
+		background-position: -320px -32px;
+	}
+
+	.af-fileicon-tex {
+		background-position: -352px -32px;
+	}
+
+	.af-fileicon-tgz {
+		background-position: -384px -32px;
+	}
+
+	.af-fileicon-txt {
+		background-position: -416px -32px;
+	}
+
+	.af-fileicon-vcf {
+		background-position: -448px -32px;
+	}
+
+	.af-fileicon-video {
+		background-position: -480px -32px;
+	}
+
+	.af-fileicon-vsd {
+		background-position: -512px -32px;
+	}
+
+	.af-fileicon-wav {
+		background-position: -544px -32px;
+	}
+
+	.af-fileicon-wma {
+		background-position: -576px -32px;
+	}
+
+	.af-fileicon-wmv {
+		background-position: -608px -32px;
+	}
+
+	.af-fileicon-xls,
+	.af-fileicon-xlsm,
+	.af-fileicon-xlsx {
+		background-position: -640px -32px;
+	}
+
+	.af-fileicon-xml {
+		background-position: -672px -32px;
+	}
+
+	.af-fileicon-xpi {
+		background-position: -704px -32px;
+	}
+
+	.af-fileicon-xvid {
+		background-position: -736px -32px;
+	}
+
+	.af-fileicon-zip {
+		background-position: -768px -32px;
+	}
+
+/* /36 */
+
+
+/* 37 -> */
+
+
+/* 38 Security Grid */
```

Comment:
should these be moved to controlpanel.css ?

Reply:
I think you are right. I'll do that.

Reply:
corrected, see last commit

Reply:
> should these be moved to controlpanel.css ?

@Timo-Breumelhof  Oops. This is actually used in module settings when mode is social group rather than standard. I am moving back. Sorry !!

Reply:
ha ha no problem, this is exactly why I added the numbers.. :-)